### PR TITLE
Test/menu structure docs

### PR DESCRIPTION
**Pull request by Ghislain89**
This PR adds 

- A test to read the directory structure of the pages dir for the docs app and validate if each markdown file is represented by a page in the menu structure.
- Cleans up old Cypress dependencies and tests

### CODE REVIEW

File: packages/apps/docs/src/components/Layout/components/TreeMenu/Item.tsx [hunk 1]
```diff
@@ -25,7 +25,11 @@ export const Item: FC<IItem> = ({ item, level }) => {
   );
 
   return (
-    <li className={classNames(listItemClass, listItemVariants[`l${level}`])}>
+    //TODO: Check with Steven for a good way to do this.
```
Comment (Ghislain89) [inline packages/apps/docs/src/components/Layout/components/TreeMenu/Item.tsx:28]: @sstraatemans I had to add test-id here to be able to identify the element, eslint wasn't too happy about this. Would you be able to support with a slight alternative approach that does not piss off eslint?

File: packages/apps/docs/src/components/Layout/components/TreeMenu/MainTreeItem.tsx [hunk 2]
```diff
@@ -65,7 +66,9 @@ export const MainTreeItem: FC<IProps> = ({ item, root = false, level = 1 }) => {
         </>
       )}
       {!root && hasSubmenu ? (
-        <li key={item.root} ref={ref}>
+        //TODO: Check with Steven for a good way to do this.
+        // eslint-disable-next-line react/no-unknown-property
```
Comment (Ghislain89) [inline packages/apps/docs/src/components/Layout/components/TreeMenu/MainTreeItem.tsx:70]: @sstraatemans same here.

File: packages/apps/docs/src/components/Layout/components/TreeMenu/Item.tsx (hunk 1 above)
Comment (realdreamer) [reply packages/apps/docs/src/components/Layout/components/TreeMenu/Item.tsx:28]: how about using the `data-testid`? data attributes won't trigger any eslint warnings

Comment (Ghislain89) [reply packages/apps/docs/src/components/Layout/components/TreeMenu/Item.tsx:28]: You're right. Thanks mo! <3

File: .changeset/config.json [hunk 3]
```diff
@@ -9,5 +9,5 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": ["@kadena/bootstrap-lib", "@kadena/tools"]
+  "ignore": ["@kadena/bootstrap-lib", "@kadena/tools", "@kadena-dev/e2e-tests", "@kadena/docs"]
```
Comment (sstraatemans) [inline .changeset/config.json:12]: 🔥

**Review by sstraatemans — APPROVED** [review]

**Review by realdreamer — APPROVED** [review]

**Pull request merged**